lib/core/lib/ops/organizationOps.js

```diff
@@ -297,6 +357,172 @@ const organizationOps = {
+      tasks.push(insertOrgPolicies)
+    }
+
+    db.withTransaction(tasks, (err, res) => {
+      if (err) {
+        return cb(err)
+      }
+
+      organizationOps.readById(id, cb)
+    })
+  },
+
+  /**
+   * Add one or more policies to an organization
+   *
+   * @param  {Object}   params { id, policies }
+   * @param  {Function} cb
+   */
+  addOrganizationPolicies: function addOrganizationPolicies (params, cb) {
+    const { id, policies } = params
+    if (policies.length <= 0) {
+      return organizationOps.readById(id, cb)
+    }
+
+    const tasks = [
+      (job, next) => {
+        Joi.validate({ id, policies }, validationRules.addOrganizationPolicies, (err) => {
+          if (err) return next(Boom.badRequest(err))
+
+          next()
+        })
+      },
+      (job, next) => {
+        job.id = id
+        job.policies = policies
+
+        next()
+      },
+      checkOrg,
+      (job, next) => {
+        utils.checkPoliciesOrg(job.client, job.policies, job.id, next)
+      },
+      insertOrgPolicies
+    ]
+
+    db.withTransaction(tasks, (err, res) => {
+      if (err) return cb(err)
+
+      organizationOps.readById(id, cb)
+    })
+  },
+
+  /**
+   * Rmove all organization's attached policies
+   *
+   * @param  {Object}   params { id, organizationId }
+   * @param  {Function} cb
+   */
+  deleteOrganizationAttachedPolicies: function deleteOrganizationAttachedPolicies (params, cb) {
+    const { id } = params
+    const tasks = [
+      (job, next) => {
+        Joi.validate({ id }, validationRules.deleteOrganizationPolicies, (err) => {
+          if (err) return next(Boom.badRequest(err))
+
+          next()
+        })
+      },
+      (job, next) => {
+        job.id = id
+
+        next()
+      },
+      checkOrg,
+      clearOrganizationAttachedPolicies
+    ]
+
+    db.withTransaction(tasks, (err, res) => {
+      if (err) {
+        return cb(err)
+      }
+
+      organizationOps.readById(id, cb)
+    })
+  },
+
+  /**
+   * Remove one organization policy
+   *
+   * @param  {Object}   params { id, policyId }
+   * @param  {Function} cb
+   */
+  deleteOrganizationAttachedPolicy: function deleteOrganizationAttachedPolicy (params, cb) {
+    const { id, policyId } = params
+    const tasks = [
+      (job, next) => {
+        Joi.validate({ id, policyId }, validationRules.deleteOrganizationPolicy, (err) => {
+          if (err) return next(Boom.badRequest(err))
+
+          next()
+        })
+      },
+      (job, next) => {
+        job.id = id
+        job.policyId = policyId
+
+        next()
+      },
+      checkOrg,
+      clearOrganizationAttachedPolicy
+    ]
+
+    db.withTransaction(tasks, (err, res) => {
+      if (err) {
+        return cb(err)
+      }
+
+      organizationOps.readById(id, cb)
+    })
+  },
+
+  insertPolicies: function insertPolicies (client, id, policies, cb) {
+    const sqlQuery = SQL`
+      INSERT INTO organization_policies (
+        policy_id, org_id
+      ) VALUES
```

Comment:
I think we need to add the [`ON CONFLICT` clause](https://github.com/nearform/labs-authorization/blob/master/lib/core/lib/ops/teamOps.js#L133)